test/core/channel/BUILD

```diff
@@ -87,7 +87,6 @@ grpc_cc_test(
     external_deps = [
         "gtest",
     ],
-    flaky = True,  # TODO(b/151792070)
```

Comment:
you'll need to regenerate projects after this change.
Have you confirmed locally that the test now passes?

Reply:
Yeah I confirmed it passes but other test started failing, though.